magefiles/magefile.go

```diff
@@ -485,6 +485,9 @@ func setRequiredEnvVars() error {
 				envVarPrefix = "MULTI_PLATFORM_CONTROLLER"
 				imageTagSuffix = "multi-platform-controller"
 				testSuiteLabel = "multi-platform"
+				im := strings.Split(os.Getenv("OTP_SERVER_IMAGE"), "@")
+				os.Setenv("MULTI_PLATFORM_CONTROLLER_OTP_IMAGE_REPO", im[0])
+				os.Setenv("MULTI_PLATFORM_CONTROLLER_OTP_IMAGE_TAG", im[1])
```

Comment:
[this line](https://github.com/redhat-appstudio/e2e-tests/pull/1134/files#diff-d7a1577b2d546e34aaf816bdd1b83edef28a66a555f049767bafa83268b7efbbR358) needs to change to
```
if strings.HasSuffix(jobName, "-service-e2e") || strings.Contains(jobName, "controller-e2e") {
```

otherwise we'll never get to this code block in CI